README.md

```diff
@@ -133,6 +133,12 @@ Opens a given note in your `$EDITOR`. Name can be an absolute path, or a relativ
 
 If no file-suffix is given in `note-name`, the notes will attempt to open `note-name.md` (or whatever your default suffix is set to). However, if the note-name is given an suffix, the default suffix will not be appended (e.g. `notes open note-name.txt` will open `note-name.txt`; not `note-name.md` or `note-name.txt.md`).
 
+### `notes append <note-name> [message]`
+
+Appends a given note with the test `message` from the command line. If no note yet exists, a new note of <note-name> will be created. This command also accepts stdin
```

Comment:
A couple of small typos here. I think this should be "Appends **to** a given note with the **text** `message`".

I'm not sure we should say "from the command line" - everything here is from the command line really! I guess you mean without opening the editor?

How about we simplify the start of this and use something like

> Appends `message` to the `note-name` note. If this note does not exist...

Reply:
I did mean without opening the editor. Do you have any suggestions how to word it so it doesn't sound so weird?